tuf/client/updater.py

```diff
@@ -2721,20 +2842,15 @@ def download_target(self, target, destination_directory):
                                target_filepath.lstrip(os.sep))
     destination = os.path.abspath(destination)
     target_dirpath = os.path.dirname(destination)
-    if target_dirpath:
-      try:
-        os.makedirs(target_dirpath)
-      
-      except OSError as e:
-        if e.errno == errno.EEXIST:
-          pass
-        
-        else:
-          raise
     
-    else:
-      message = repr(target_dirpath) + ' does not exist.'
-      logger.warning(message)
-      raise tuf.Error(message)
+    try:
+      os.makedirs(target_dirpath)
+    
+    except OSError as e:
+      if e.errno == errno.EEXIST:
```

Comment:
Not quite resolved in 321: see [this comment](https://github.com/theupdateframework/tuf/pull/321/files#r54270274)